schema_editor/test/spec/schemas/schemas-service.spec.js

```diff
@@ -148,4 +176,69 @@ describe('ase.schemas:Schemas', function () {
             message: jasmine.stringMatching('Diff')
         }]));
     });
+
+    // NOTE: This test must be updated when new field types are added.
+    it('should preserve all information when serializing and deserializing schemas', function () {
+        // In practice this means that the serialize and deserialize methods are inverses
```

Comment:
Something like this post wouldn't work?

http://stackoverflow.com/questions/17370427/loading-a-mock-json-file-within-karmaangularjs-test